wordpress-mobile/WordPress-iOS
# Stats Revamp: Guides for Totals cells

Fixes #18867. This PR adds guide cards for the new Totals cells in stats.

**In app:**

|   |   |
|---|---|
| <img width="350" src="https://user-images.githubusercontent.com/4780/174052905-b5009b64-a6a3-4b0d-a5e3-1e38ccd03a52.png"> | <img width="350" src="https://user-images.githubusercontent.com/4780/174052916-913969e5-6480-42cc-8b02-6539a284ed04.png"> |

**Design:**

<img width="350" alt="Screenshot 2022-06-16 at 11 40 16" src="https://user-images.githubusercontent.com/4780/174053092-cc2d49e2-5ea2-4b80-a517-30204441c89b.png">

**Logic:**

* Likes: If the user has published a post, show info about that post
* Comments: Always show
* Followers: Follower count is < 2 (site can have a follower count of 1 if the user is following their own site)

**Known issues**

* There's currently no navigation to the latest post by tapping on the link in the Likes card. I'll add this in a followup.
* The Likes guide doesn't refresh the text colour if there's a change to / from dark mode. You have to navigate away from and back to the screen to see this update.

**To test**

* Build and run, and navigate to a site with no likes, no published posts, no comments, and no followers.
* You should see the comments and followers guides.
* Navigate to a site with > 1 follower. You should not see the followers guide.
* Navigate to a site with a published post. You should see the Likes guide.

## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
3. What automated tests I added (or what prevented me from doing so)

N/A

PR submission checklist:
